crates/biome_css_analyze/src/utils.rs

```diff
@@ -92,3 +93,8 @@ pub fn find_font_family(value: CssGenericComponentValueList) -> Vec<AnyCssValue>
     }
     font_families
 }
+
+pub fn strip_vendor_prefix(property: &str) -> String {
+    let re = Regex::new(r"^-\w+-").unwrap();
+    re.replace(property, "").to_string()
+}
```

Comment:
I think we aren't ready yet to handle prefixes in this way:
- I don't think we need a regex, the cases are very simple that we can just list the prefixes in an ordered array, and check against them using the binary search
- regex are expensive, here we **create** a new regex every time we call the function, which is very inefficient. You should create it as a `const`
- for now we can just return a `bool` saying if the function contains a prefix, and inside the rule we check `endsWith("image-set")`. There's no need to allocate a new string we don't use it

Reply:
I have modified it to check directly within the `run` method using the `ends_with` method.